test/unit/assorted/uri_options.spec.test.ts

```diff
@@ -0,0 +1,61 @@
+import { loadSpecTests } from '../../spec';
+import { executeUriValidationTest } from '../../tools/uri_spec_runner';
+
+describe('URI option spec tests', function () {
+  const suites = loadSpecTests('uri-options');
+
+  const skipTests = [
+    // TODO: fix?
+    'directConnection=true with multiple seeds',
+    'loadBalanced=true with directConnection=false causes an error',
+
+    // TODO:? serverSelectionTryOnce is not implemented, should it be?
+    'Valid options specific to single-threaded drivers are parsed correctly',
+
+    // TODO?: need to implement test to have the files in the correct place in order to make sure it doesn't throw
```

Comment:
TODO: make test pass for now
TODO: follow up ticket to change the behavior to read file async, may involve new props